gremlin-python/src/main/python/gremlin_python/driver/aiohttp/transport.py

```diff
@@ -16,12 +16,11 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-
-
 import aiohttp
 import asyncio
 import async_timeout
 from aiohttp import ClientResponseError
+import logging
```

Comment:
It's cool that we have logging now. This is going to be a huge help. Elsewhere I'd commented that we use:

https://docs.python.org/3/library/warnings.html

in one place: `graph.py`. I believe there is a way to re-route warnings to the logger:

https://docs.python.org/3/library/logging.html#logging.captureWarnings

perhaps we should do that if you haven't already somewhere? or perhaps we don't need `warnings` anymore at all? or perhaps we should double down on usage? I'll leave it to you to make a choice on how to rectify `warnings` and `logging` but i think it would be good to have a strategy here.

Reply:
I read around online and the general consensus from what I read is that warnings.warn is the correct move in the case of a deprecated API, so I will use that as is and add it to submitAsync as suggested.

Reply:
Will keep warnings.warn as is because it is good practice for deprecated API's from what I have read online, but will use logging from now on to log other things that are not API deprecations.